src/handlers/deeplinks.ts

```diff
@@ -123,7 +122,24 @@ export default async function handleDeeplink(
         }
         break;
       }
+      case 'f2c': {
+        logger.log('Handling F2C deeplink', { url });
+
+        const { provider, sessionId } = urlObj.query;
+
+        if (!provider || !sessionId) {
+          logger.warn('Received FWC deeplink with invalid params', {
+            query: urlObj.query,
+          });
+        }
```

Comment:
Other providers might strip these, honestly not sure, but seemed like a good thing to track.